src/components/helpers/Markdown.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <div class="vsd-markdown">
+        <section class="page-section" id="markdown">
+            <div class="container">
+                <VueShowdown
+                :markdown='markdown'
+                flavor="github"
+                :options="{ emoji: true }"/> 
+            </div>
+        </section>
+    </div>
+</template>
+
+<script>
+import { VueShowdown } from 'vue-showdown'
+
+export default {
+    name: 'vsd-markdown',
+    props: {
+        url: {
+            type: String
+        }
+    },
+    components: {
+        VueShowdown
+    },
+    data: () => ({
+        markdown: ''
+    }),
+    async mounted () {
+        await fetch(this.url)
+            .then(res => {
+                    return res.text()
+            })
+            .then(text => {
+                this.markdown = text
```

Comment:
Since we are using `await` and a promise chain, we can actually do something like:
```suggestion
this.markdown = await fetch(this.url).then(res => res.text()).catch(err => {
  return `# Unable to fetch
    ${err}`
})
```

Reply:
But actually, this is only relevant if we keep the `fetch`